_posts/en/newsletters/2023-03-08-newsletter.md

```diff
@@ -0,0 +1,205 @@
+---
+title: 'Bitcoin Optech Newsletter #241'
+permalink: /en/newsletters/2023/03/08/
+name: 2023-03-08-newsletter
+slug: 2023-03-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Alternative design for OP_VAULT:** Greg Sanders [posted][sanders
+  vault] to the Bitcoin-Dev mailing list an alternative design for
+  providing the features of the `OP_VAULT`/`OP_UNVAULT` proposal (see
+  [Newsletter #234][news234 vault]).  His alternative would add three
+  opcodes instead of two.  To provide an example:
+
+    - *Alice deposits funds in a vault* by paying a [taproot][topic
+      taproot] tree which contains at least two [tapscripts][topic
+      tapscript], one with her unvault tapscript and one with her freeze
+      tapscript, e.g. `tr(key,{trigger,freeze})`.
```

Comment:
Sorry, opinionated on the terminology here. ;)
I would prefer “(taproot) script tree” over “taproot tree”. I would prefer referring to the scripts in the script tree as “leafscripts”, from what I understand “Tapscript” refers to the specific set of Script rules defined by BIP342 to be used in leafscripts with leaf version 0xc0, not a leafscript that uses those rules.

```suggestion
    - *Alice deposits funds in a vault* by paying a P2TR output with a [script tree][topic
      taproot] which contains at least two [leafscripts][topic
      tapscript], one with her unvault leafscript and one with her freeze
      leafscript, e.g. `tr(key,{trigger,freeze})`.
```

Reply:
> I would prefer “(taproot) script tree” over “taproot tree”.

I'm fine with this.

> I would prefer referring to the scripts in the script tree as “leafscripts”, from what I understand “Tapscript” refers to the specific set of Script rules defined by BIP342 to be used in leafscripts with leaf version 0xc0, not a leafscript that uses those rules.

I can't think of a good argument against this (and, yeah, I tried :-), so I'll make these changes, but since you have a strong opinion about this, would you consider opening a PR to edit https://bitcoinops.org/en/topics/tapscript/ to include a description of the differences between tapscript and leafscript (and adding an alias in the YAML so that we get leafscript in our topics index)?  If you're willing, you can do this at your leisure; it's not blocking on merging this newsletter.  Please CC me if you open a PR as I don't subscribe to all PRs on the repo.